news/_posts/translation/2019-10-09-do-you-speak-prestashop-september-2019.md

```diff
@@ -0,0 +1,122 @@
+
+
+### :muscle: Go full 1.7! 
+
+Since version is not maintained by PrestaShop anymore (read [this post](http://build.prestashop.com/news/1.6.1.x-what-s-next) to learn more about it if you missed the information), we feel that it will be more relevant to only mention the 1.7 in those reports from now on. So let’s all head to our latest version and be proud of it, there is so much to do!
+
+
+### :two_hearts: Special thanks to newcomers
+
+Many people subscribe to the project(s) of their choice each month, but not so many start translating straightaway. For that reason, we would like to thank the ones that kind of rush into the effort immediately, aha! So lot of love to the dedicated Adib Aroui (aroui.adib), Chay Bourngern (benjarongmart), dress-safe, Efren Morales Martinez (EfrenMorales), Rolf Fuchs (higgs77), iansyhau, and lutek. See you soon on Crowdin… and PrestaShop. ;-)
+
+
+## A few stats
+ 
+* **30 members** were active on the project this month.
+* A total of **10,171 words have been translated** and **7,778 validated**.
+* All this in **26 different languages**.
+ 
+Thank you for your involvement!
+ 
+
+## Top contributors
+ 
+A lot of you are working every day on Crowdin to have PrestaShop available in many languages, and we can't thank you enough for your dedication! Here are the **most active translators and proofreaders** for September 2019.
+
+#### Top 10 translators in September:
+ 
+| |Translator | Language | # Words
+|-|---------- | -------- | ----------------
+ 1. | Rolf Fuchs (higgs77) | German | 1,566
+ 2. | Stamatis (breezer) | Greek | 1, 549
+ 3. | SeongHyeon Cho (jaymz9634) | Korean | 1,415
+ 4. | Ramya Shantha Peiris (Ramzhead8899) | Sinhala | 1,359
+ 5. | Gabriel Tenita (ggedamed) | Romanian | 1,109
+ 6. | Tantely R. (rabtaray) | Malagasy | 723
+ 7. | xaero| Dutch | 334
+ 8. | Henry Wong (monkeyjr) | Chinese Traditional | 285
+ 9. | Sretko Devič (Chico) | Slovenian | 248
+10. | lutek | Polish | 239
+ 
+ 
+#### Top 10 proofreaders in September:
+ 
+| | Proofreader | Language | # Words
+|-| ---------- | -------- | ----------------
+ 1. | Tantely R. (rabtaray) | Malagasy | 1,632
+ 2. | Stamatis (breezer) | Greek | 1,549
+ 3. | SeongHyeon Cho (jaymz9634) | Korean | 1,390
+ 4. | Gabriel Tenita (ggedamed) | Romanian | 1,124
+ 5. | Rijad Osmanovic (rijado) | Bosnian | 471
+ 6. | Cha (cafetango) | Chinese Traditional | 372
+ 7. | Stijn (stijn.bosman) | Dutch | 324
+ 8. | Sretko Devič (Chico) | Slovenian | 249
+ 9. | DaoKakao | Russian | 197
+10. | Zoran Tejic (zotamal) | Serbian (Latin) | 196
+
+Congrats, and welcome to our new top contributors!
+ 
+Remember, you can see who's been contributing to our translation project thanks to the [Translators page](http://translators.prestashop.com/).
+ 
+ 
+## Complete translations
+ 
+#### Fully translated languages
+ 
+At the end of September 2019, PrestaShop 1.7.6 was fully available (= 100% translated and validated) in **8 languages**:
+ 
+| Czech | English | **Greek** | French |
```

Comment:
It's strange, you talk about 8 languages but you show only 1 🤔?

Reply:
Ah no I got it 😄the way it's displayed as a table it a bit weird

Reply:
Only one?